main.tf

```diff
@@ -269,6 +269,7 @@ module "cache" {
 
   create_cache_bucket     = var.cache_bucket["create"]
   cache_bucket_prefix     = var.cache_bucket_prefix
+  cache_bucket_name_include_account_id = var.cache_bucket_name_include_account_id
```

Comment:
Can you run terraform fmt on `main.tf` this will format the code according hashicorp standards. CI build checks for this.

Reply:
Formatting fixed